here-naksha-app-service/src/test/java/com/here/naksha/app/service/EventHandlerApiTest.java

```diff
@@ -288,4 +288,76 @@ void tc0162_testUpdateEventHandlerWithMismatchingId() throws Exception {
         .hasStreamIdHeader(streamId)
         .hasJsonBody(expectedErrorResponse);
   }
+
+  @Test
+  void tc0180_testDeleteHandler() throws Exception {
+    // Test API : DELETE /hub/handlers/{handlerId}
+    // Given:
+    final String createHandlerJson = loadFileOrFail("EventHandlerApi/TC0180_deleteHandler/create_handler.json");
+    final String streamId = UUID.randomUUID().toString();
+    final HttpResponse<String> createResponse =
+            getNakshaClient().post("hub/handlers", createHandlerJson, streamId);
+    assertThat(createResponse).hasStatus(200);
+    // When:
+    final HttpResponse<String> response =
+            getNakshaClient().delete("hub/handlers/handler-to-delete", streamId);
+
+    // Then:
+    assertThat(response)
+            .hasStatus(200)
+            .hasStreamIdHeader(getHeader(response, HDR_STREAM_ID));
+
+    final HttpResponse<String> getResponse =
+            getNakshaClient().get("hub/handlers/handler-to-delete", streamId);
+    assertThat(getResponse)
+            .hasStatus(404)
+            .hasStreamIdHeader(getHeader(response, HDR_STREAM_ID));
+  }
+
+  @Test
+  void tc0181_testDeleteHandlerInUse() throws Exception {
+    // Test API : DELETE /hub/handlers/{handlerId}
+    // Given:
+    final String streamId = UUID.randomUUID().toString();
+    final String createHandlerJson = loadFileOrFail("EventHandlerApi/TC0181_deleteHandlerInUse/create_handler.json");
```

Comment:
We can use standard functions here, which do the similar thing:
CommonApiTestSetup.createHandler()
CommonApiTestSetup.createSpace()